packages/flutter_test/lib/src/controller.dart

```diff
@@ -781,6 +781,14 @@ abstract class WidgetController {
     ).view;
   }
 
+  FlutterView? _maybeViewOf(finders.FinderBase<Element> finder) {
+    try {
+      return _viewOf(finder);
```

Comment:
this should be the other way around

_viewOf should use _maybeViewOf and throw if it returns null (if we really want it to throw)